src/crimson/net/Protocol.h

```diff
@@ -11,6 +11,11 @@
 #include "Fwd.h"
 #include "SocketConnection.h"
 
+// TODO: CEPH_MSGR2_FEATURE_COMPRESSION
+#define CEPH_MSGR2_SUPPORTED_FEATURES_CRIMSON \
+        (CEPH_MSGR2_FEATURE_REVISION_1 | \
+         0ULL)
```

Comment:
i'd suggest move this definition to the `.cc` file. and define it like
```c++
// TODO: CEPH_MSGR2_FEATURE_COMPRESSION
static const uint64_t CEPH_MSGR2_SUPPORTED_FEATURES_CRIMSON = 
  (CEPH_MSGR2_FEATURE_REVISION_1 |
   CEPH_MSGR2_FEATURE_COMPRESSION |
   UINT64_C(0));
```

for better type safety. and  an `unsigned long long` is not necessary `uint64_t`, `UINT64_C(0)` is.

Reply:
Working on it.

Reply:
done.